### DISCUSSION TITLE
QGIS plugin UMEP

### DISCUSSION THREAD

**Discussion by Mengyaru1029** [UMEP-dev/UMEP#468] [Q&A] [answered]
QGIS plugin UMEP calculate the average radiation temperature height is the default of 1.1 meters, where you can modify the height parameter

**Comment by biglimp** [comment, marked as answer]
The height parameter is not displayed as a parameter that you can change, you can only change the posture of the body (sitting, standing). If you like to change the height you need to change the code. You can do this in solweig.py line 710. There you see height that can be set for standing or sitting persons

**Reply by ML426** [reply]
Hi,
I have a similar question.
If I modify the height parameter, is it possible to install the modified version on Qgis or to use it directly in python?
If so, how can I do it in either case ?

Best regard, 
ML

**Reply by biglimp** [reply]
Change the value in line 710 in solweig.py, then restart QGIS and the new value will be used.

**Reply by ML426** [reply]
Should I edit on the local folder or on the one on the online directory?
For now, I use UMEP installed from the online plugin available on Qgis.
Can you please provide more details on the procedure?

**Reply by biglimp** [reply]
locate your local plugin folder. This varies depending on your system platfrom. If you are using windows, it is something like: C:\Users\your_username\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins
